tests/integration/pilot/vm/legacy_os_vm_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright Istio Authors
```

Comment:
given this is called "legacy_os_vm_test.go" I would also expect there to be a "vm_test.go" or something. Its a bit weird to me that the "legacy" test includes testing Debian 10 for example - maybe we should just call this vm_test.go?

Reply:
I will update the name. Thanks!